reana_server/reana_admin.py

```diff
@@ -250,3 +251,44 @@ def token_revoke(id, email, admin_access_token):
     # send notification to user by email
     click.secho(f'User\'s {user_id} ({user_email}) token successfully revoked',
                 fg='green')
+
+
+@reana_admin.command(help='Get a status report of the REANA system.')
+@click.option(
+    '--from',
+    'from_',
+    type=click.DateTime(),
+    help='Since wich point in time to retrieve the information from?')
+@click.option(
+    '--until',
+    type=click.DateTime(),
+    help='Since wich point in time to retrieve the information from?')
+@click.option(
+    '--type',
+    'types_',
+    multiple=True,
+    default='all',
+    type=click.Choice(list(STATUS_OBJECT_TYPES.keys()) + ['all'],
+                      case_sensitive=False),
+    help='Type of information to be displayed?')
+@click.option(
+    '--format',
+    default='txt',
+    type=click.Choice(['txt']),
+    help='Type of information to be displayed?')
+@admin_access_token_option
+def status_report(from_, until, types_, format, admin_access_token):
+    """Retrieve a status report summary of the REANA system."""
+    def _format_statuses(type_, stats):
+        """Format stats dictionary object."""
+        click.echo(type_.upper())
+        for stat_name, stat_value in stats.items():
+            click.echo(f'{stat_name}: {stat_value}')
+
+    types_ = STATUS_OBJECT_TYPES.keys() if 'all' in types_ else types_
+    for type_ in types_:
+        statuses_obj = STATUS_OBJECT_TYPES[type_](
+            from_=from_, until=until)
```

Comment:
How can we pass a user?

Reply:
I didn't pass it yet as it is not implemented. Same happens with `from_` and `until` which only work for one of the statuses...